src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -462,7 +462,7 @@ public static class Parser
                                                                 string.Empty,
                                                                 RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
-        private static readonly RegexReplace WebsitePrefixRegex = new RegexReplace(@"^\[\s*[-a-z]+(\.[a-z]+)+\s*\][- ]*|^www\.[a-z]+\.(?:com|net|org)[ -]*",
+        private static readonly RegexReplace WebsitePrefixRegex = new RegexReplace(@"^\[\s*[-a-z]+(\.[a-z]+)+\s*\][- ]*|^www\.[a-z]+(\.[a-z]+)[ -]*",
```

Comment:
Are there actually single character TLDs?

Reply:
RFC-952 allows it, but there currently doesn't seem to be any and there is some debate in recent RFC proposals.. Also, they can be alpha numeric but can't be all numeric, and must be under 24 chars. 

```
A "name" (Net, Host, Gateway, or Domain name) is a text string up
to 24 characters drawn from the alphabet (A-Z), digits (0-9), minus
sign (-), and period (.).
```

```
There is an additional rule that essentially requires
     that top-level domain names not be all-numeric.
```

REF current list: https://data.iana.org/TLD/tlds-alpha-by-domain.txt

Maybe `^\[\s*[-a-z]+(\.[a-z]+)+\s*\][- ]*|^www\.[a-z]+(\.(?![0-9]*$)[a-z0-9]{2,24})[ -]*`